### PR TITLE
Fix the QPACK offline interop tool qif.

### PR DESCRIPTION
**Pull request by stammw**
The [bin] entry has been commented so it was left behind and couldn't run.

### CODE REVIEW

**Comment by Ralith** [comment]
Yeah, I think this would be significantly simplified by just embracing anyhow's dynamically dispatched error type. There's not much value to an error enum that will never, ever be matched on.

**Comment by stammw** [comment]
I left that because I was pretty sure `Error` was also used for logic. I doesn't seem to be the case. 

Here, using an enum as still one goal: reduce call site error message duplication.

I'll rewrite all this.

**Comment by stammw** [comment]
Got rid of `err_derive` and `enum Error`. It's actually much better now :)

I also added a small doc to get started with the tool.

File: interop/Cargo.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ publish = false
 [dependencies]
 anyhow = "1.0.22"
 bytes = "0.5.2"
+err-derive = "0.2.3"
```
Comment (djc) [inline interop/Cargo.toml:12]: Not needed anymore?

Comment (stammw) [reply interop/Cargo.toml]: oops...

File: interop/src/qif.rs [hunk 2]
```diff
@@ -365,6 +400,7 @@ impl EncodedFile {
             stream_count += 1;
         }
         fs::write(&self.file, buf)?;
+        println!("encoded in {:?}", self.file);
```
Comment (djc) [inline interop/src/qif.rs:403]: This doesn't seem to belong?

Comment (stammw) [reply interop/src/qif.rs:403]: It actually is. When you run it, you'll want to know what happened. I figured this out writing the doc.

Comment (djc) [reply interop/src/qif.rs:403]: Fair enough!

**Pull request merged**